<launch>

  <arg name="joy_dev" default="/dev/input/js0" />
  <arg name="joy_autorepeat_rate" default="20" />
  <arg name="joy_deadzone" default="0.05" />

  <node pkg="joy" type="joy_node" name="joy_node">
    <param name="dev" value="$(arg joy_dev)" />
    <param name="deadzone" value="$(arg joy_deadzone)" />
    <param name="autorepeat_rate" value="$(arg joy_autorepeat_rate)" />
  </node>

  <rosparam command="load" file ="$(find oxar_is_bored)/config/button_map_xbox.yaml"/>

  <node pkg="oxar_is_bored" type="teleop_arm.py" name="teleop_node" output="screen">
    <rosparam param="button_mappings">$(arg button_map_xbox)</rosparam>
    <remap from="safety_stop" to="safety_stop" />
  </node>

    <!--Move to arm launch on zaltec pc?-->

    <!--Launches rosserial to comunicate with arduino-->
  <!--node pkg="rosserial_python" type="serial_node.py" name="serial_node">
    <param name="port" value="/dev/ttyACM0"/>
    <param name="baud" value="115200"/>
  </node-->
  

</launch>
